litmus/AArch64Compile_litmus.ml

```diff
@@ -1449,15 +1462,28 @@ module Make(V:Constant.S)(C:Config) =
     | I_STOPBH (op,v,w,rs,rn) ->
         stop (stopbh_memo op v w) V32 rs rn::k
 (* Conditional selection *)
-    | I_CSEL (v,r,ZR,ZR,c,Inc) ->
-        let o,f = match v with
-        | V32 -> arg1 "wzr" (fun s -> "^wo"^s) r
-        | V64 -> arg1 "xzr" (fun s -> "^o"^s) r
-        | V128 -> assert false
-        and t = match v with V32 -> word | V64 -> quad | V128 -> assert false in
+    | I_CSEL (v,r,ZR,ZR,c,(Inc|Inv as op)) ->
+        let o,f = arg1o v r
+        and t =
+          match v with V32 -> word | V64 -> quad | V128 -> assert false in
         let memo =
-          sprintf "cset %s,%s" f (pp_cond (inverse_cond c)) in
+          let op =
+            match op with
+            | Inc -> "cset"
+            | Inv -> "csetm"
+            | _ -> assert false in
+          sprintf "cset %s,%s,%s" op f (pp_cond (inverse_cond c)) in
```

Comment:
Is the syntax really: "cset cset,xzr,ne"?

```suggestion
          sprintf "%s %s,%s,%s" op f f (pp_cond (inverse_cond c)) in
```

Reply:
Nice catch. The syntax is `cset _r_,_cond_` I fix it independently. Thanks.